### PR TITLE
Fluentd elasticsearch support and small fixes.

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@DTadrzak  please review

File: stackanetes-services/fluentd/fluentd-elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: fluentd-elasticsearch
+name: fluentd-elasticsearch
+command: "/run.sh"
+files: []
```
Comment (DTadrzak) [inline stackanetes-services/fluentd/fluentd-elasticsearch.yml]: if you don't need property "files" just remove it

File: stackanetes/common/utils.py [hunk 2]
```diff
@@ -38,9 +38,6 @@ def get_kubectl_command(add_namespace=True):
     if CONF.stackanetes.context:
         context = "--context=" + CONF.stackanetes.context
         cmd.append(context)
-    if add_namespace and CONF.stackanetes.namespace:
-        namespace = "--namespace=" + CONF.stackanetes.namespace
```
Comment (DTadrzak) [inline stackanetes/common/utils.py]: if namespace is no longer need please remove add_namespace flag from method

**Comment by DTadrzak** [comment]
lgtm

**Pull request merged**